test/forge/ReallocateWithdrawTest.sol

```diff
@@ -104,6 +115,36 @@ contract ReallocateWithdrawTest is IntegrationTest {
         assertApproxEqAbs(vault.idle(), expectedIdle, 1, "vault.idle() 1");
     }
 
+    function testReallocateWithdrawIncreaseSupply(uint256[3] memory assets) public {
+        _setCap(allMarkets[2], 3 * CAP2);
+
+        allocations.push(MarketAllocation(allMarkets[0], 0));
+        allocations.push(MarketAllocation(allMarkets[1], 0));
+        allocations.push(MarketAllocation(allMarkets[2], 2 * CAP2));
+
+        vm.startPrank(ALLOCATOR);
```

Comment:
Ideally we only prank the `reallocate` call otherwise this PR is neat!

Reply:
Did you move it right before `reallocate`?

Reply:
Fixed. ty